shared/helpers/view.js

```diff
@@ -67,7 +67,13 @@ function getClientPlaceholder(viewName, viewOptions) {
 
   // create a list of data attributes
   var attrString = _.inject(viewOptions, function(memo, value, key) {
-    if (_.isArray(value) || _.isObject(value)) { value = JSON.stringify(value); }
+    if (_.isArray(value) || _.isObject(value)) {
+      if (key === '_block' && value.contructor === Handlebars.SafeString) {
```

Comment:
typo of `constructor`? why not just say `value instance of Handlebars.SafeString`?